File: _persist/temp/venda/Java/Pessoa.java
package MORM.Java.Models;

import java.util.Date;
import MORM.Java.Classes;

@Tabela("PESSOA")
public class Pessoa extends CollectionItem
{
    @Campo("ID_PESSOA", CampoTipo.tfKey)
    public String Id_Pessoa;
    @Campo("U_VERSION", CampoTipo.tfNul)
    public String U_Version;
    @Campo("CD_OPERADOR", CampoTipo.tfReq)
    public Integer Cd_Operador;
    @Campo("DT_CADASTRO", CampoTipo.tfReq)
    public DateTime Dt_Cadastro;
    @Campo("CD_PESSOA", CampoTipo.tfReq)
    public Integer Cd_Pessoa;
    @Campo("NM_PESSOA", CampoTipo.tfReq)
    public String Nm_Pessoa;
    @Campo("NR_CPFCNPJ", CampoTipo.tfReq)
    public String Nr_Cpfcnpj;
    @Campo("NR_RGIE", CampoTipo.tfReq)
    public String Nr_Rgie;
    @Campo("NM_FANTASIA", CampoTipo.tfNul)
    public String Nm_Fantasia;
    @Campo("CD_CEP", CampoTipo.tfReq)
    public Integer Cd_Cep;
    @Campo("NM_LOGRADOURO", CampoTipo.tfReq)
    public String Nm_Logradouro;
    @Campo("NR_LOGRADOURO", CampoTipo.tfReq)
    public String Nr_Logradouro;
    @Campo("DS_BAIRRO", CampoTipo.tfReq)
    public String Ds_Bairro;
    @Campo("DS_COMPLEMENTO", CampoTipo.tfNul)
    public String Ds_Complemento;
    @Campo("CD_MUNICIPIO", CampoTipo.tfReq)
    public Integer Cd_Municipio;
    @Campo("DS_MUNICIPIO", CampoTipo.tfReq)
    public String Ds_Municipio;
    @Campo("CD_ESTADO", CampoTipo.tfReq)
    public Integer Cd_Estado;
    @Campo("DS_ESTADO", CampoTipo.tfReq)
    public String Ds_Estado;
    @Campo("DS_SIGLAESTADO", CampoTipo.tfReq)
    public String Ds_Siglaestado;
    @Campo("CD_PAIS", CampoTipo.tfReq)
    public Integer Cd_Pais;
    @Campo("DS_PAIS", CampoTipo.tfReq)
    public String Ds_Pais;
    @Campo("DS_FONE", CampoTipo.tfNul)
    public String Ds_Fone;
    @Campo("DS_CELULAR", CampoTipo.tfNul)
    public String Ds_Celular;
    @Campo("DS_EMAIL", CampoTipo.tfNul)
    public String Ds_Email;
    @Campo("IN_CONSUMIDORFINAL", CampoTipo.tfNul)
    public String In_Consumidorfinal;
}

File: _persist/temp/venda/CSharp/Pessoa.cs
using MORM.CSharp.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("PESSOA")]
    public class Pessoa : CollectionItem
    {
        [Campo("ID_PESSOA", CampoTipo.tfKey)]
        public string Id_Pessoa { get; set; }
        [Campo("U_VERSION", CampoTipo.tfNul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.tfReq)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.tfReq)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("CD_PESSOA", CampoTipo.tfReq)]
        public int Cd_Pessoa { get; set; }
        [Campo("NM_PESSOA", CampoTipo.tfReq)]
        public string Nm_Pessoa { get; set; }
        [Campo("NR_CPFCNPJ", CampoTipo.tfReq)]
        public string Nr_Cpfcnpj { get; set; }
        [Campo("NR_RGIE", CampoTipo.tfReq)]
        public string Nr_Rgie { get; set; }
        [Campo("NM_FANTASIA", CampoTipo.tfNul)]
        public string Nm_Fantasia { get; set; }
        [Campo("CD_CEP", CampoTipo.tfReq)]
        public int Cd_Cep { get; set; }
        [Campo("NM_LOGRADOURO", CampoTipo.tfReq)]
        public string Nm_Logradouro { get; set; }
        [Campo("NR_LOGRADOURO", CampoTipo.tfReq)]
        public string Nr_Logradouro { get; set; }
        [Campo("DS_BAIRRO", CampoTipo.tfReq)]
        public string Ds_Bairro { get; set; }
        [Campo("DS_COMPLEMENTO", CampoTipo.tfNul)]
        public string Ds_Complemento { get; set; }
        [Campo("CD_MUNICIPIO", CampoTipo.tfReq)]
        public int Cd_Municipio { get; set; }
        [Campo("DS_MUNICIPIO", CampoTipo.tfReq)]
        public string Ds_Municipio { get; set; }
        [Campo("CD_ESTADO", CampoTipo.tfReq)]
        public int Cd_Estado { get; set; }
        [Campo("DS_ESTADO", CampoTipo.tfReq)]
        public string Ds_Estado { get; set; }
        [Campo("DS_SIGLAESTADO", CampoTipo.tfReq)]
        public string Ds_Siglaestado { get; set; }
        [Campo("CD_PAIS", CampoTipo.tfReq)]
        public int Cd_Pais { get; set; }
        [Campo("DS_PAIS", CampoTipo.tfReq)]
        public string Ds_Pais { get; set; }
        [Campo("DS_FONE", CampoTipo.tfNul)]
        public string Ds_Fone { get; set; }
        [Campo("DS_CELULAR", CampoTipo.tfNul)]
        public string Ds_Celular { get; set; }
        [Campo("DS_EMAIL", CampoTipo.tfNul)]
        public string Ds_Email { get; set; }
        [Campo("IN_CONSUMIDORFINAL", CampoTipo.tfNul)]
        public string In_Consumidorfinal { get; set; }
    }
}